src/pages/ReimbursementAccount/BankInfo/BankInfo.tsx

```diff
@@ -118,7 +118,7 @@ function BankInfo({reimbursementAccount, reimbursementAccountDraft, plaidLinkTok
                     [BANK_INFO_STEP_KEYS.ROUTING_NUMBER]: '',
                     [BANK_INFO_STEP_KEYS.ACCOUNT_NUMBER]: '',
                     [BANK_INFO_STEP_KEYS.PLAID_MASK]: '',
-                    [BANK_INFO_STEP_KEYS.IS_SAVINGS]: '',
+                    [BANK_INFO_STEP_KEYS.IS_SAVINGS]: false,
```

Comment:
why is this change needed?

Reply:
Before migration we had incorrect type value here. After migration `typecheck` throws type mismatch so it had to be changed. Since empty string is falsy I assume using `false` boolean value is fine here.